sshtunnel.py

```diff
@@ -174,7 +167,7 @@ def check_addresses(address_list):
         check_address(address)
 
 
-def create_logger(logger=None, loglevel=DEFAULT_LOGLEVEL):
+def create_logger(logger=None, loglevel=None):
```

Comment:
More preferably, in this case explicitly pass an argument specifying the default log level.